website/docs/components/time/partials/specifications/anatomy.md

```diff
@@ -0,0 +1,15 @@
+## Anatomy
+
+<!-- image then table -->
```

Comment:
Issue: needs to be done

Reply:
I'm thinking we may want to remove this tab...

Reply:
I went ahead and removed the tab as it doesn't seem relevant. (Following what we did for the `Text` component.)